dds/Cargo.toml

```diff
@@ -16,9 +16,10 @@ keywords = ["dds", "rtps", "middleware", "network", "udp"]
 categories = ["api-bindings", "network-programming"]
 
 [dependencies]
-dust_dds_derive = { path = "../dds_derive", version = "0.3" }
+dust_dds_derive = { path = "../dds_derive", version = "0.4" }
 serde = { version = "=1.0", default-features = false, features = ["derive"] }
 cdr = "=0.2.4"
+serde-xml-rs = "0.6.0"

```

Comment:
where is that used?

Reply:
This is the serializer and deserializer for the XML representation of the type (in type_support.rs)